.github/workflows/create-release-pr.yml

```diff
@@ -0,0 +1,65 @@
+name: Create Release PR
+on: 
+  workflow_dispatch:
+    inputs:
+      new_version:
+        description: "New sdk version"
+        required: true
+jobs:
+  Release:
+    runs-on: ubuntu-latest
+    permissions:
+        contents: write
+        pull-requests: write
+    steps:
+    - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
+      with: 
+        ref: ${{ github.head_ref }}
+    - name: validate package dependencies
+      run: |
+        RESULT=$(swift package show-dependencies | grep unspecified)
+        if [ -n "$RESULT" ]; then
+          echo "Unspecified dependencies found in the package:"
+          echo "$RESULT"
+          exit 1
+        else
+          echo "All dependencies are specified correctly."
+        fi
+    - name: update version file
+      run: |
+        echo '// Copyright © 2025 Elasticsearch BV
+        //
+        //   Licensed under the Apache License, Version 2.0 (the "License");
+        //   you may not use this file except in compliance with the License.
+        //   You may obtain a copy of the License at
+        //
+        //       http://www.apache.org/licenses/LICENSE-2.0
+        //
+        //   Unless required by applicable law or agreed to in writing, software
+        //   distributed under the License is distributed on an "AS IS" BASIS,
+        //   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+        //   See the License for the specific language governing permissions and
+        //   limitations under the License.
+
+        import Foundation
+
+        extension ElasticApmAgent {
+            public static let elasticSwiftAgentVersion: String = "${{ inputs.new_version }}"
+        ' > Sources/apm-agent-ios/Version.swift  
+    - name: update docs versions
+      run: | 
```

Comment:
Captures the otel version used in Package.swift and sets that in the supported technologies doc